tests/test_tictactoe.py

```diff
@@ -10,6 +10,23 @@ def tt():
     return tictactoe.TicTacToeEnv()
 
 
+def test_policy_page_lines():
+    """
+    Just a calling a policy to make sure that it replay from a couple of different boards.
+    """
+    game = tictactoe.TicTacToeEnv()
+    while not game.done:
+        game.step(random.choice(game.valid_actions))
+        if not game.done:
+            game.step(tictactoe.policy_page_lines(game._get_obs()))
+
+    game = tictactoe.TicTacToeEnv()
+    while not game.done:
+        game.step(tictactoe.policy_page_lines(game._get_obs()))
+        if not game.done:
+            game.step(random.choice(game.valid_actions))
```

Comment:
issue: there aren't any asserts or anything here - I guess this tests that the `policy_page_lines` always returns a valid move. 
suggestion:This could be tested by just getting the move and checking it's in `game.valid_actions`

suggestion: It'd be good doing a little check that given an empty board, it returns (0,0) and that a partially filled board, it does return the next move as expected

Reply:
Yes, I think i'd assumed that if it didn't get a valid move the error that that threw woud be enough to fail the test, and provide a good explanation when it did so.  Maybe that isn't safe?
I'll check again.  Breaking it down into some more specific asserts might be sensible.

Reply:
Yes, we'll revisit.